crd/cluster.go

```diff
@@ -0,0 +1,25 @@
+package crd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/grtl/mysql-operator/util"
+)
+
+func MySQLClusterCreateCRD() error {
+	gopath := os.Getenv("GOPATH")
+	stderr, err := util.Create(gopath + "/src/github.com/grtl/mysql-operator/artifacts/cluster-crd.yaml")
+
+	if strings.Contains(stderr.String(), "AlreadyExists") {
+		fmt.Println("Custom resource definition already exists")
```

Comment:
Do we really want to be printing to stdout from a utility function?

Reply:
You should return `AlreadyExists` error to be handled later (see `NewAlreadyExists` from package `"k8s.io/apimachinery/pkg/api/errors"`).